docs/report/report.tex

```diff
@@ -385,14 +385,60 @@ \subsubsection{Retrospektive}
+Im Studium wurde den Studierenden der Vorteil von Versionskontrollsystemen wie Git vermittelt, doch fand sich im Studium selten ein praktisches Anwendungszenario, dass Git über die Hauptfunktion der Versionierung hinaus sinnvoll genutzt hat.
+Durch dieses Projekt fand sich das erste mal im Studium eine sinnvolle Anwendung, wie in größeren Projekten oder in der Industrie auch mit Pull Requests und Code Reviews zu arbeiten.
+Dadurch traten mehrere Effekte auf:
+Wenn mehrere Teammitglieder über den Code anderer schauten, fielen umso wahrscheinlicher Fehler auf und eine insgesamt bessere Codequalität wurde sichergestellt.
+Code Reviews hatten außerdem den Effekt, dass Teammitglieder neue Funktionen und Änderungen kennenlernen, verstehen und gegebenenfalls auch direkt Fragen dazu stellen konnten.
+Sich an diesen Deployment Cycle zu gewöhnen brauchte eine gewisse Zeit, aber fand von Sprint zu Sprint immer mehr Wertschätzung im Team.
+
+\subsubsection{Automatisierte Tests und Deployment}
+Jeder Pull Request unterlief automatisch auch mehrere automatisierte Tests, welche mit eigenen Testparametern über den neuen Quellcode lief.
+Durch diese Tests wurden sowohl dem Team viel manuelle Arbeit abgenommen, als auch wurde die Fehlerquelle Mensch minimiert.
+Neue Änderungen im Master wurden automatisch auf Github.io veröffentlicht, als auch wurden die \textit{LaTeX} Dateien als PDF im eigenen Dokumente Branch gebaut.
+Für Pull Requests wurde auch die jeweilige Version gebaut und auf einer \colorbox{yellow}{temporären} Webseite veröffentlicht.
+Dadurch konnte sich das Team mehr auf das Weiterentwickeln des Projektes konzentrieren.
+
+\subsubsection{Regelmäßige Standups und Retrospektiven}
+Durch die regelmäßigen Standups entstand ein Ort für das gegenseitige ansprechen von Problemen, updaten von Fortschritten und neuen Funktionen.
+So gingen seltener Probleme unter, neues Wissen wurde schneller verbreitet und die gesamte Teamproduktivität wurde verbessert.
+Das bei den Retrospektiven entstandene Feedback zwischen dem Teammitgliedern war sehr wichtig für das Teamklima.
+Auch die hierbei angesprochenen Probleme und Verbesserungsvorschläge wurden als sehr wertvoll vom Team für die nächsten Sprints erkannt.
+
+\subsubsection{Kommunikation und Eigeninitiative}
+Die Standups hatten auch den Effekt, dass das Team anfangs sich zu sehr auf diesen Kommunikationsweg verlassen hat, wodurch Probleme und wichtige Kommunikation bei Abhängigkeit zwischen Tickets zu selten statt fand.
+Hier fehlte anfangs die Eigeninitiative, bei Problemen aktiv sich Hilfe zu holen, auch weil manche Tickets zu schwierig wirkten, um diese alleine zu bewältigen.
+Hierbei halfen einerseits die Retrospektiven, die dieses Problem ansprachen, aber vor allem die teaminterne Evaluation.
+
+\subsubsection{Planung, Zielsetzung und Einschätzung}
+Das größte Problem während des Projektes war, dass gegen Ende eines Sprints geplante Tickets nur teilweise fertiggestellt wurden.
```

Comment:
```suggestion
Das größte Problem während des Projektes war, dass gegen Ende eines Sprints geplante Tickets teilweise nicht fertiggestellt wurden.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/419/commits/c5dae29b261f99d39aac66c889e6bb96205a2f1a